test/tools/ossfuzz/const_opt_ossfuzz.cpp

```diff
@@ -0,0 +1,8 @@
+#include <test/tools/fuzzer_common.h>
+
+extern "C" int LLVMFuzzerTestOneInput(uint8_t const* _data, size_t _size)
+{
+    string input(reinterpret_cast<char const*>(_data), _size);
+    FuzzerUtil::testConstantOptimizer(input, true);
```

Comment:
Do we really want it to be quiet?

Reply:
Yes, because at the moment "quietness" only decides whether or not to print on cout and we do not want the fuzzing binary to be overloading the fuzzing log with debug messages.

However. for reproducing reported bugs, we would have to set `quiet=false`, right?